Mention in docs that `--project .` is needed in some cases

In complex setups, PK plugin cannot be found by mvn without `--projects .` option.
One example is https://github.com/exasol/spark-connector

Command `mvn project-keeper:verify` ends up with error
```
[ERROR] No plugin found for prefix 'project-keeper' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/Users/maxim.lapan/.m2/repository), central (https://repo.maven.apache.org/maven2)] -> [Help 1]
```

But command `mvn project-keeper:verify --projects .` works fine.
This should be reflected in User's guide, as the mvn error is confusing.

--- a/README.md
+++ b/README.md
@@ -171,6 +171,19 @@ In addition this plugin can also fix the project structure. For that use:
 mvn project-keeper:fix
 ```
 
+For multi-module projects these commands may fail with the following error:
+
+```
+[ERROR] No plugin found for prefix 'project-keeper' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/home/user/.m2/repository), central (https://repo.maven.apache.org/maven2)] -> [Help 1]
+```
+
+In this case add command line option `--projects .`:
+
+```sh
+mvn project-keeper:verify --projects .
+mvn project-keeper:fix --projects .
+```
+
 You can skip the execution of project-keeper by adding `-Dproject-keeper.skip=true` to your maven command.
 
 ### Standalone Command Line Interface

--- a/README.md
+++ b/README.md
@@ -161,13 +161,13 @@ The verification is bound to the maven `package` lifecycle phase. So it is autom
 
 You can also run the checks manually using:
 
-```shell script
+```sh
 mvn project-keeper:verify
 ```
 
 In addition this plugin can also fix the project structure. For that use:
 
-```shell script
+```sh
 mvn project-keeper:fix
 ```
 
@@ -192,14 +192,14 @@ Use the `project-keeper-cli` for analyzing non-Maven projects like Golang.
 
 Run the following commands to verify a project:
 
-```shell
+```sh
 cd path/to/project
 java -jar path/to/project-keeper-cli-2.7.1.jar verify
 ```
 
 Run the following commands to fix the project structure:
 
-```shell
+```sh
 cd path/to/project
 java -jar path/to/project-keeper-cli-2.7.1.jar fix
 ```

--- a/doc/changes/changelog.md
+++ b/doc/changes/changelog.md
@@ -1,5 +1,6 @@
 # Changes
 
+* [2.9.15](changes_2.9.15.md)
 * [2.9.14](changes_2.9.14.md)
 * [2.9.13](changes_2.9.13.md)
 * [2.9.12](changes_2.9.12.md)

--- a/doc/changes/changes_2.9.15.md
+++ b/doc/changes/changes_2.9.15.md
@@ -4,9 +4,9 @@ Code name:
 
 ## Summary
 
-## Features
+## Documentation
 
-* ISSUE_NUMBER: description
+* #485: Added note about `--projects .` command line option to user guide
 
 ## Dependency Updates
 

--- a/doc/changes/changes_2.9.15.md
+++ b/doc/changes/changes_2.9.15.md
@@ -1,0 +1,53 @@
+# Project Keeper 2.9.15, released 2023-??-??
+
+Code name:
+
+## Summary
+
+## Features
+
+* ISSUE_NUMBER: description
+
+## Dependency Updates
+
+### Project Keeper Core
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-model-classes:2.9.14` to `2.9.15`
+
+#### Runtime Dependency Updates
+
+* Updated `com.exasol:project-keeper-java-project-crawler:2.9.14` to `2.9.15`
+
+#### Test Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-test-setup:2.9.14` to `2.9.15`
+
+### Project Keeper Command Line Interface
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-core:2.9.14` to `2.9.15`
+
+#### Test Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-test-setup:2.9.14` to `2.9.15`
+
+### Project Keeper Maven Plugin
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-core:2.9.14` to `2.9.15`
+
+### Project Keeper Java Project Crawler
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-model-classes:2.9.14` to `2.9.15`
+
+### Project Keeper Shared Test Setup
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-model-classes:2.9.14` to `2.9.15`

--- a/doc/developers_guide/developers_guide.md
+++ b/doc/developers_guide/developers_guide.md
@@ -48,7 +48,7 @@ Usually you need to
 
 In some cases PK runs maven as a shell command. Maven will then search your for PK module "project crawler" in the version identical to PK opened in your IDE. The following command publishes project crawler to your local maven repository `~/.m2`:
 
-```shell
+```sh
 mvn clean install -DskipTests
 ```
 

--- a/parent-pom/pom.xml
+++ b/parent-pom/pom.xml
@@ -28,7 +28,7 @@
         </repository>
     </distributionManagement>
     <properties>
-        <revision>2.9.14</revision>
+        <revision>2.9.15</revision>
         <maven.version>3.9.5</maven.version>
         <minimum.maven.version>3.6.3</minimum.maven.version>
         <junit.version>5.10.0</junit.version>

